client/public/locales/es/translation.json

```diff
@@ -77,23 +77,23 @@
   "Desktop": "Escritorio",
   "Layout": "Diseño",
   "Always visible": "Siempre visible",
-  "Save my settings (in the browser store)": "Guardar mis ajustes (en el almacenamiento del navegador)",
-  "<p>Game rules determine if it is allowed to skip levels and if the games runs checks to only allow unlocked tactics and theorems in proofs.</p><1>Note: \"Unlocked\" tactics (or theorems) are determined by two things: The set of minimal tactics needed to solve a level, plus any tactics you unlocked in another level. That means if you unlock <1>simp</1> in a level, you can use it henceforth in any level.</1><p>The options are:</p>": "<p>Las reglas del juego determinan si se permite saltarse niveles y si el juego realiza comprobaciones para permitir únicamente tácticas y teoremas desbloqueados en las pruebas.</p><1>Nota: las tácticas (o teoremas) \"Desbloqueadas\" está determinadas por dos cosas: el conjunto mínimo de tácticas necesarias para resolver un nivel, más cualquier táctica que hayas desbloqueado en otro nivel. Esto significa que si desbloqueas <1>simp</1> en un nivel, puedes usarlo a partir de entonces en cualquier nivel.</1><p>Las opciones son:</p>",
+  "Save my settings (in the browser store)": "Guardar mis ajustes (en el navegador)",
```

Comment:
"in the storage of the browser" is a bit redundant, you can just say "in the browser".